pvlib/clearsky.py

```diff
@@ -782,147 +1029,13 @@ def rmse(alpha):
 
     if return_components:
         components = OrderedDict()
-        components['mean_diff_flag'] = c1
-        components['max_diff_flag'] = c2
-        components['line_length_flag'] = c3
-        components['slope_nstd_flag'] = c4
-        components['slope_max_flag'] = c5
-        components['mean_nan_flag'] = c6
-        components['windows'] = clear_windows
-
-        components['mean_diff'] = np.abs(meas_mean - alpha * clear_mean)
-        components['max_diff'] = np.abs(meas_max - alpha * clear_max)
-        components['line_length'] = meas_line_length - clear_line_length
-        components['slope_nstd'] = meas_slope_nstd
-        components['slope_max'] = (np.max(
-            meas_slope - alpha * clear_slope, axis=0))
-
+        components['mean_diff'] = c1
+        components['max_diff'] = c2
+        components['line_length'] = c3
+        components['slope_nstd'] = c4
+        components['slope_max'] = c5
+        components['mean_nan'] = c6
+#        components['windows'] = clear_windows
```

Comment:
If some keys are disappearing from this dict we should consider mentioning it in the whatsnew

Reply:
That is unintentional, at one point I was thinking that `clear_windows` would go away, but it isn't. I'll include it in `components`.